build_deploy.sh

```diff
@@ -12,15 +26,32 @@ login_container_registry() {
     local PASSWORD="$2"
     local REGISTRY="$3"
 
-    $CONTAINER_ENGINE_CMD login "-u=${USER}" "-p=${PASSWORD}" "$REGISTRY"
+    container_engine_cmd login "-u=${USER}" "-p=${PASSWORD}" "$REGISTRY"
 }
 
 check_required_registry_credentials() {
 
-    if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" || -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
-        echo "QUAY_USER, QUAY_TOKEN, RH_REGISTRY_USER and RH_REGISTRY_TOKEN must be set"
+
+    if [[ -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
+        echo "RH_REGISTRY_USER and RH_REGISTRY_TOKEN must be set"
         exit 1
     fi
+
+    if ! "$LOCAL_BUILD"; then
```

Comment:
Wouldn't this run a command? Also please check other usages.

```
if [[ ! "$LOCAL_BUILD" ]]; then
```

Reply:
refactored